### PR TITLE
minor changes to the banhammer

### PR DESCRIPTION
**Pull request by Spacedong**
gave it a better description and upped the volume of the bwoink sound to scare people more

### CODE REVIEW

**Comment by Pyko1** [comment]
Should be "BWOINK! Hammer" at desc

**Comment by McBawbaggings** [comment]
Okay add yangelog please

**Comment by McBawbaggings** [comment]
+1

**Comment by Spacedong** [comment]
the changelog is there, remove that tag

**Comment by McBawbaggings** [comment]
Changelog is .txt and not .yml

**Comment by ArcLumin** [comment]
Code's good, changelog's not. Fix that shit.

**Comment by blobbernaut** [comment]
change of the year please merge

**Comment by Pyko1** [comment]
No merging until the changelog is fixed.

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/banhammer
-	desc = "A banhammer"
+	desc = "<font color='red'><b>BWOINK<b></font>"
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/weaponry.dm]: Incorrect terminating bold tag

File: html/changelogs/Spacedong - banhammertweak.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+################################
```
Comment (JohnGinnane) [inline html/changelogs/Spacedong - banhammertweak.txt]: Wrong file type

Comment (ArcLumin) [reply html/changelogs/Spacedong - banhammertweak.txt]: spess donk.txt

**Comment by SpankMasterson** [comment]
+1 if dong addresses requested reviews

**Comment by Spacedong** [comment]
changelog is now .yml

**Comment by Spacedong** [comment]
aka ready to merge now

**Comment by Pyko1** [comment]
approved but apparently needs dongcode changes

**Comment by McBawbaggings** [comment]
Dongo too lazy to fix the HTML tag.

**Comment by Pyko1** [comment]
bawws code seems fine, merging

**Pull request merged**